images/macos/tests/Ruby.arm64.Tests.ps1

```diff
@@ -3,7 +3,7 @@ Import-Module "$PSScriptRoot/../helpers/Tests.Helpers.psm1" -DisableNameChecking
 
 $os = Get-OSVersion
 
-Describe "Ruby" -Skip:($os.IsVentura -or $os.IsBigSur -or $os.IsMonterey -or $os.IsSonoma) {
+Describe "Ruby" -Skip:($os.IsBigSur -or $os.IsMonterey -or $os.IsVenturaX64 -or $os.IsSonomaX64) {
```

Comment:
What do you think about to use new `IsArm64` here? Since those test for all arm64 based OSs.